src/elements/content-sharing/ContentSharing.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * @flow
+ * @file ContentSharing Element
+ * @author Box
+ */
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
I've found it easier to mock and spy on these helpers when they're attached to a namespace:

```
import * as React from 'react';

React.useState<Whatever | null>(null);
```